.github/workflows/iam-drift-detection.yml

```diff
@@ -0,0 +1,69 @@
+# Workflow for IAM drift detection
+# Approach:
+#   Run on schedule or manual
+#   Execute drift script
+#   If drift, create issue
+
+name: 'IAM drift detection'
+on:
+  workflow_call:
+    inputs:
+      organization-id:
```

Comment:
nit: prefer underscores for naming here `organization_id`

Reply:
Done.